src/BizHawk.Emulation.Cores/Computers/DOS/DOSBox.cs

```diff
@@ -66,7 +67,7 @@ public DOSBox(CoreLoadParameters<object, SyncSettings> lp)
 			// Parsing rom files
 			foreach (var file in _roms)
 			{
-				var ext = file.RomPath.Substring(startIndex: file.RomPath.LastIndexOf('.')).ToLowerInvariant();
+				var ext = file.RomPath.Substring(startIndex: file.RomPath.LastIndexOf('.')).ToLowerInvariant().RemovePrefix('.');

```

Comment:
```suggestion
				var ext = file.RomPath.Substring(startIndex: file.RomPath.LastIndexOf('.') + 1).ToLowerInvariant();
```
My bad there

Reply:
Actually what happens if the file has no extension?

Reply:
I guess this would treat the whole string as the extension, but that's a weird edge case to handle here. I don't think the file would ever reach the core ctor.

Reply:
Can't you guys just use `Path.GetExtension` please?

Reply:
fixed